### DISCUSSION TITLE
Jellyfin community container, which data is borged?

### DISCUSSION THREAD

**Discussion by b9AcE** [nextcloud/all-in-one#4446] [Questions] [answered]
The readme.md for the [Jellyfin community container](https://github.com/nextcloud/all-in-one/tree/main/community-containers/jellyfin ) states that "The data of Jellyfin will be automatically included in AIO's backup solution" but I wonder if by "data" we mean only configuration-data but not the media-data, or both?
I believe many will potentially find a backup done of the media-data of potentially many terabytes to be prohibitively excessive redundant and storage space expensive, especially as it may be either already being backed up by a previously existing solution or deemed such that it is preferable to just reacquire the media-data if need be rather than keeping double or even triple copies of the media-data.
**IF** it currently is mandatory both conf and media, maybe there should be some configuration parameter implemented enabling each site to choose whether to exclude that part or not?

I have obviously not just tried running it before asking those who might know,
as I don't want to accidentally wreck something through overfilling backup or such.

**Comment by szaimen** [comment, marked as answer]
Regarding jellyfin, only its configuration data is included in the backup

**Reply by Anvil5465** [reply]
Does this still apply if the /media data is stored in a Nextcloud user's files?

Edit: answer is no https://github.com/nextcloud/all-in-one/discussions/5893
